Core/Enumerations/SpellTags.cs

```diff
@@ -0,0 +1,128 @@
+﻿// <copyright file="SpellTags.cs" company="LeagueSharp">
+//    Copyright (c) 2015 LeagueSharp.
+// 
+//    This program is free software: you can redistribute it and/or modify
+//    it under the terms of the GNU General Public License as published by
+//    the Free Software Foundation, either version 3 of the License, or
+//    (at your option) any later version.
+// 
+//    This program is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//    GNU General Public License for more details.
+// 
+//    You should have received a copy of the GNU General Public License
+//    along with this program.  If not, see http://www.gnu.org/licenses/
+// </copyright>
+
+namespace LeagueSharp.SDK.Core.Enumerations
+{
+    /// <summary>
+    /// Properties that a spell can have
+    /// </summary>
+    public enum SpellTags
+    {
+        /// <summary>
+        /// The spell deals damage
+        /// </summary>
+        Damage,
+
+        /// <summary>
+        /// The spell's effects are AoE
+        /// </summary>
+        AoE,
+
+        /// <summary>
+        /// The spell applies on-hit effects.
+        /// </summary>
+        AppliesOnHitEffects,
+        
+        /// <summary>
+        /// The spell applies CC
+        /// </summary>
+        CrowdControl,
+
+        /// <summary>
+        /// The spell applies a shield on the target
+        /// </summary>
+        Shield,
+        
+        /// <summary>
+        /// The spell can heal
+        /// </summary>
+        Heal,
+
+        /// <summary>
+        /// The spell makes the target enter a stasis state (invulnerable)
+        /// </summary>
+        Stasis,
```

Comment:
I am not sure what level of detail you are going for here so my question might be irrelevant but what about Kayle's R (invulnerable), Fizz's E (untargetable), and Olaf's R (immune to CC)? If you mean Stasis as in Hourglass those three I mentioned are different and don't have a tag.